src/utils/enum-utilities.ts

```diff
@@ -0,0 +1,15 @@
+import { FilingTypes } from '@bcrs-shared-components/enums'
+
+// NB: These two methods are just for getPnedingCoa() in useBcrosFilings store
+// more methods to be added. Or the EnumUtilities class can be replaced by functions
```

Comment:
I would update this comment to make these more generic if necessary (i.e. to pass in the desired filing type or filing status rather than have separate functions for every possibility). Also the name 'EnumUtilities' is a bit misleading I think - could we change that to something like 'FilingUtil' as this will always be specific to filings data I think?

Reply:
actually is there a reason this isn't in the filing store? It seems intuitive to have something like isFilingType(types: FilingTypes[]) / isStatus(status: FilingStatusE[]) in the filing store, similar to what I was asking for in the business store?